pkg/policyfilter/k8s_test.go

```diff
@@ -44,11 +44,11 @@ import (
 
 type tlog struct {
 	*testing.T
-	Log *logrus.Logger
+	Logger *logrus.Logger
 }
 
 func (tl tlog) Write(p []byte) (n int, err error) {
-	tl.Logf((string)(p))
+	tl.Log(string(p))
```

Comment:
Hey @kkourt, I think it was a mistake here that maybe you wanted to use Logrus here and not the native testing `Log` functions. I also think that it doesn't really matter maybe? What do you think?

Reply:
I honestly don't remember, but change LGTM :)